### ISSUE TITLE
Authentication Popup on first load

### ISSUE THREAD

**Issue by juaorpo** [FelisCatus/SwitchyOmega#1481]
Hi, 

I have set a HTTP proxy with username and password using the settings interface, saved, and applied changes succesfully. The connection works, but opon browser startup, the authentication is not automatic or silent, it requires user action. After startup I receive a pop up with a user/pass promt with the values pre-filled. I can type it manually or even cancel it and reload the tab, both options works, switchy authenticates using the settings information and I will be able to surf. 

In summary, auth settings work. I can connect through the proxy. The only problem is the first load, where I am prompted to type in user/password with a pop-up, which I can even cancel, but I need to click something.

This is incovenient, specially for unattended environments. The pop up requires to be cancelled or the pasword to be manually input, it requires at least human interaction. 

Can this be solved?

**Comment by FelisCatus** [comment]
This happens because some requests are made before SwitchyOmega is fully initialized.

If I understand correctly, this is something that we cannot solve on Chrome. However, once we implement #1456, we may have a way to solve this on Firefox.
